nativelib/src/main/scala/scala/scalanative/native/system.scala

```diff
@@ -0,0 +1,39 @@
+package scala.scalanative.native
+
+object system {
```

Comment:
We already have C API for setenv/getenv and Java API, I don't think that adding yet another one is a good idea here.

Reply:
The idea here was to provide some wrapper so we could setenv without a Zone etc. I know this may be a bigger issue but I think at some point we may need to support common functions via easy to use Scala interfaces.

Reply:
It's not pulling it's weight unfortunately, especially given that there is a Java API for this.

Reply:
Ok, this is not the best code and creating an API is a big deal - it was just added to eliminate using `Zone` in the test. Not sure to which Java API you are referring.